packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -120,56 +148,32 @@ export function PanelChrome({
         ) : null}
       </div>
 
-      <div className={cx(styles.headerContainer, dragClass)} style={headerStyles} data-testid="header-container">
-        {title && (
-          <h6 title={title} className={styles.title}>
-            {title}
-          </h6>
-        )}
+      {(hoverHeader || !hasHeader) && menu && (
+        <HoverWidget menu={menu} title={title} offset={hoverHeaderOffset} dragClass={dragClass}>
+          {headerContent}
+        </HoverWidget>
+      )}
 
-        <PanelDescription description={description} className={dragClassCancel} />
+      {hasHeader && (
+        <div className={cx(styles.headerContainer, dragClass)} style={headerStyles} data-testid="header-container">
+          {headerContent}
 
-        {titleItems.length > 0 && (
-          <div className={cx(styles.titleItems, dragClassCancel)} data-testid="title-items-container">
-            {titleItems.map((item) => item)}
+          <div className={styles.rightAligned}>
+            {menu && <PanelMenu menu={menu} title={title} menuButtonClass={cx(styles.menuItem, 'show-on-hover')} />}
```

Comment:
Got confused a bit by this, but I might have missed this in design. Isn't the hover header supposed to hide all the information, and enclose it to be visible only on... hover?

With empty title, but any other info (i.e. description or streaming indicator) I can still see the header:

<img width="1140" alt="image" src="https://user-images.githubusercontent.com/2376619/216071966-e8bf775c-2056-4a87-a30c-e93cc0824635.png">

Reply:
Yeah, that's what we were discussing today even as well because it's a bit confusing.. So, on the one hand, we want to have a separate control over whether or not the panel has a header. This would be perfect with "header yes?" _toggle_ like the one for transparency. However, this is heavily a breaking change actually and should probably be introduced in Grafana 10?
On the other hand, the current functionality is indeed that removing the title of a panel makes it to be without a header. In this sense we should indeed cover the current functionality. So imho it seems like `PanelStateWrapper` can send down to `PanelChrome` the prop `hoverHeader={true}` if there is no `title` for the panel, no other conditions (that I know of (?)).

Reply:
I think we are not passing down in the `PanelStateWrapper` the `hoverHeader` prop, it should be true if  `title` is empty. It will look like this:

![image](https://user-images.githubusercontent.com/239999/216079032-bd1982ab-f93c-4bb4-bf04-50bacbd21fb2.png)


In the future(after the first iteration), we want to add a configurable prop in the panel to display the header on hover or not

![image](https://user-images.githubusercontent.com/239999/216079483-0d957307-a2bc-466e-87e9-74bf32a037bf.png)